### PR TITLE
test: Use the new placement annotation

### PR DESCRIPTION
**Pull request by astefanutti**
This adapts the e2e tests and covers the happy path for the use cases introduced by #41.

### CODE REVIEW

File: e2e/ingress_test.go [hunk 1]
```diff
@@ -138,32 +129,58 @@ func TestIngress(t *testing.T) {
...
+	})))
+
+	gracePeriodDuration := deleteDelay.TTLDefault - 10*time.Second // take some slack
+	cluster1LabelSelector := ClusterLabel + "=" + cluster1.Name
+
+	// Check the shadow Ingress, Service and Deployment for cluster 1 are not deleted before the grace period expires
+	test.Consistently(getShadowResources(test, namespace, cluster1LabelSelector), gracePeriodDuration).Should(HaveLen(3))
+
+	// Finally, check the shadow Ingress, Service and Deployment for cluster 1 are deleted, once the grace period has expired
+	test.Eventually(getShadowResources(test, namespace, cluster1LabelSelector)).Should(BeEmpty())
```
Comment (philbrookes) [inline e2e/ingress_test.go:165]: Is it worth also deleting the root items and seeing that everything cleans up? In this scenario it should not wait for the grace period either.

**Comment by philbrookes** [comment]
LGTM

**Review by philbrookes — APPROVED** [review]

File: e2e/ingress_test.go (hunk 1 above)
Comment (astefanutti) [reply e2e/ingress_test.go:165]: Good idea, added in fcfab96ba26c1f38093efc0ccf092fefa3feee4f.

**Pull request merged**